package.json

```diff
@@ -144,10 +154,27 @@
                     "group": "2_functionAppControlCommands@3"
                 },
                 {
-                    "command": "azureFunctions.deployZip",
+                    "command": "azureFunctions.deployZipFromExplorer",
                     "when": "view == azureFunctionsExplorer && viewItem == azureFunctionsFunctionApp",
                     "group": "3_functionAppDeployCommands@1"
                 }
+            ],
+            "explorer/context": [
+                {
+                    "command": "azureFunctions.deployZipFromFolder",
+                    "when": "explorerResourceIsFolder == true",
+                    "group": "zzz_@1"
+                }
+            ],
+            "commandPalette": [
+                {
+                    "command": "azureFunctions.deployZipFromExplorer",
+                    "when": "never"
```

Comment:
Why never?

Reply:
We have three separate 'deployZip' commands with a context-specific label for each. However, we only want _one_ of them to show in the command palette